app/src/main/java/com/boolder/boolder/view/detail/ProblemBSFragment.kt

```diff
@@ -173,6 +178,15 @@ class ProblemBSFragment(private val listener: BottomSheetListener) : BottomSheet
             text = listOfNotNull(steepnessText, sitStartText).joinToString(separator = " • ")
             isVisible = !text.isNullOrEmpty()
         }
+        val tickRepository: TickRepository = get()
```

Comment:
Let's avoid assigning this repository more than once in this fragment. It can be set as a class scoped value 😉

Reply:
Set as a class scoped value